planner/core/main_test.go

```diff
@@ -72,3 +73,7 @@ func GetJoinReorderData() testdata.TestData {
 func GetRuleConstantPropagationData() testdata.TestData {
 	return testDataMap["rule_constant_propagation_suite"]
 }
+
+func GetPlanStatsData() testdata.TestData {
```

Comment:
suggest them to move into casetest.

Reply:
Moved. Together with other tests in this file.